packages/integrations/stores/cockroach-db/src/cockroach-db.ts

```diff
@@ -0,0 +1,11 @@
+import { type CockroachConnectionOptions } from 'typeorm/driver/cockroachdb/CockroachConnectionOptions';
+import { ORMConfigStore, type ORMConfigStoreSharedConfiguration } from './ORM';
```

Comment:
Please align these with other integrations that base off of `typeorm`:
```suggestion
import { type CockroachConnectionOptions } from 'typeorm/driver/cockroachdb/CockroachConnectionOptions.js';
import { ORMConfigStore, type ORMConfigStoreSharedConfiguration } from '@configu/integrations/src/utils/ORM';
```

Reply:
Done